src/many/src/server/module/_7_kvstore_commands.rs

```diff
@@ -9,5 +9,5 @@ pub use put::*;
 #[many_module(name = KvStoreCommandsModule, id = 7, namespace = kvstore, many_crate = crate)]
 pub trait KvStoreCommandsModuleBackend: Send {
     fn put(&mut self, sender: &Identity, args: PutArgs) -> Result<PutReturns, ManyError>;
-    fn delete(&mut self, sender: &Identity, args: DeleteArgs) -> Result<DeleteReturn, ManyError>;
+    fn delete(&mut self, sender: &Identity, args: DeleteArgs) -> Result<DeleteReturns, ManyError>;
```

Comment:
The specification uses single `Return`, so I think we should use that in our code too.